apps/shipments/templates/email/shipment_link.mjml

```diff
@@ -0,0 +1,64 @@
+{% load mjml %}
+{% block subject %}shipchain.io - {{username}} shared a shipment details page with you.{% endblock %}
+
+{% block html_body %}
+{% mjml %}
+    <mjml>
+      <mj-body background-color="#ec7143">
+          <mj-section>
+            <mj-column>
+
+              <mj-image src="https://profiles.shipchain.io/static/profiles/shipchain-logo-2.png"></mj-image>
```

Comment:
Might be better to add these images to Transmission instead, so that they don't have to rely on profiles.